.github/workflows/increment-version-and-kick-off-sync.yml

```diff
@@ -17,7 +17,7 @@ jobs:
 
       # See https://github.com/im-open/git-version for more details around how to increment major/minor/patch through commit messages
       - name: Increment the version
-        uses: im-open/git-version-lite@v2.0.6
+        uses: im-open/git-version-lite@v2.0.8
```

Comment:
Do we not put a worklfow version on this, because the teams don't really edit this?

Reply:
Yeah this didn't have a version number because this is the one that belongs to the repo that it uses to version itself.